### PR TITLE
 Working on GSSlider 

### PR DESCRIPTION
**Pull request by singlapriyanka315**

### CODE REVIEW

**Review by geekykrishna — CHANGES_REQUESTED** [review]
It's missing storybook implementation, do add that as well.

**Comment by singlapriyanka315** [comment]
> It's missing storybook implementation, do add that as well.

Okay @Krishnagopal-Sinha .

**Review by geekykrishna — APPROVED** [review]

File: lib/src/utils/resolver.dart [hunk 1]
```diff
@@ -429,18 +429,50 @@ GSPlacements? resolvePlacementFromString(String? placement) {
   return placement != null ? placementMap[placement] : null;
 }
 
+// Map<String, GSConfigStyle>? resolveCompoundVariants(
```
Comment (geekruchika) [inline lib/src/utils/resolver.dart]: cleanup

Comment (singlapriyanka315) [reply lib/src/utils/resolver.dart]: Sure .

**Pull request merged**